include/vapor/WireFrameRenderer.h

```diff
@@ -81,17 +82,72 @@ class RENDER_API WireFrameRenderer : public Renderer
 
 	} _cacheParams;
 
+	// Helper class to keep track of which cell edges have been drawn so
+	// we can avoid duplicate draws.
+	//
+	class DrawList {
+	public:
+
+		// maxEntries is the maximum number of unique cell nodes. 
+		// maxLinesPerVertex is the *expected* max valence (degree) of any
+		// node (vertex). If a node has more than maxLinesPerVertex edges
+		// only the first maxLinesPerVertex edges will be recorded in
+		// DrawList. Hence, queries to edges with DrawList::InList will
+		// return false once maxLinesPerVertex has been exceeded
+		//
+		DrawList(GLuint maxEntries, size_t maxLinesPerVertex) :
+			_drawList(
+				maxEntries*maxLinesPerVertex, std::numeric_limits<GLuint>::max()
+			),
+			_maxEntries(maxEntries),
+			_maxLinesPerVertex(maxLinesPerVertex)
+		 {}	
+
+		bool InList(GLuint idx0, GLuint idx1) {
+			VAssert(idx0 < _maxEntries);
+			VAssert(idx1 < _maxEntries);
+
+			if (idx1 < idx0) {
+				std::swap( idx1, idx0 );
+			}
+
+			for (int i = 0; i<_maxLinesPerVertex; i++) {
+				if (_drawList[idx0*_maxLinesPerVertex+i] == idx1) {
+					return(true);
+				}
+				if (_drawList[idx0*_maxLinesPerVertex+i] == std::numeric_limits<GLuint>::max()) {
+					_drawList[idx0*_maxLinesPerVertex+i] = idx1;
+					return(false);
+				}
+			}
+			return(false);
+		}
+	private:
+		vector<GLuint> _drawList;
+		size_t _maxEntries;
+		size_t _maxLinesPerVertex;
```

Comment:
If these member variables aren't gonna change, then give them a const qualifier. 
```suggestion
        const size_t _maxEntries;
		const size_t _maxLinesPerVertex;
```